src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -979,11 +1008,174 @@ void testIsPoAttending() {
+            .otherParties(otherParties)
+            .build();
+        List<HearingChannel> result = getHearingChannel(caseData);
+        assertEquals(HearingChannel.VIDEO, result.get(0));
+    }
+
+    @DisplayName("When no parties contain Face to Face also don't contain Video but contain Telephone select Telephone")
+    @Test
+    void whenNoPartiesContainFaceToFace_alsoDoNotContainVideo_butContainTelephone_selectTelephone() {
+        List<CcdValue<OtherParty>> otherParties = new ArrayList<>();
+        otherParties.add(new CcdValue<>(OtherParty.builder()
+                                            .hearingOptions(HearingOptions.builder().wantsToAttend(YesNo.YES.getValue()).build())
+                                            .hearingSubtype(HearingSubtype.builder()
+                                                                .wantsHearingTypeTelephone(YesNo.YES.getValue())
+                                                                .hearingTelephoneNumber("03625263723")
+                                                                .build())
+                                            .id("2")
+                                            .name(Name.builder()
+                                                      .title("title")
+                                                      .firstName("first")
+                                                      .lastName("last")
+                                                      .build())
+                                            .build()));
+
+        SscsCaseData caseData = SscsCaseData.builder()
+            .appeal(Appeal.builder()
+                        .hearingOptions(HearingOptions.builder().wantsToAttend(YesNo.YES.getValue()).build())
+                        .hearingSubtype(HearingSubtype.builder()
+                                            .wantsHearingTypeTelephone(YesNo.YES.getValue())
+                                            .hearingTelephoneNumber("02123234823")
+                                            .build())
+                        .appellant(Appellant.builder()
+                                       .id("1")
+                                       .name(Name.builder()
+                                                 .title("title")
+                                                 .firstName("first")
+                                                 .lastName("last")
+                                                 .build())
+                                       .build())
+                        .build())
+            .dwpIsOfficerAttending(YesNo.NO.getValue())
+            .otherParties(otherParties)
+            .build();
+        List<HearingChannel> result = getHearingChannel(caseData);
+        assertEquals(HearingChannel.TELEPHONE, result.get(0));
+    }
+
+    @DisplayName("ifNetherPartiesHaveAnyPreferenceSelectedReturnNotAttending")
+    @Test
+    void ifNetherParties_haveAnyPreferenceSelected_returnNotAttending() {
```

Comment:
This test is confusing. All parties say they want to attend but the result is not attending? Is this correct?

Reply:
It's a fall back, So if the parties do want to attend but they have no preference selected it just goes to not attending as we would have nothing to work with to allow us to understand what they want.

Reply:
Alright, just confirming if this has been checked with the business?

Reply:
Confirmed and updated now